api/src/utils/get-default-value.ts

```diff
@@ -10,20 +10,8 @@ export default function getDefaultValue(
 ): string | boolean | number | Record<string, any> | any[] | null {
 	const type = getLocalType(column);
 
-	let defaultValue = column.default_value ?? null;
+	const defaultValue = column.default_value ?? null;
 	if (defaultValue === null) return null;
-	if (defaultValue === 'null') return null;
-	if (defaultValue === 'NULL') return null;
-
-	// Check if the default is wrapped in an extra pair of quotes, this happens in SQLite / MariaDB
-	if (
-		typeof defaultValue === 'string' &&
-		((defaultValue.startsWith(`'`) && defaultValue.endsWith(`'`)) ||
-			(defaultValue.startsWith(`"`) && defaultValue.endsWith(`"`)))
-	) {
```

Comment:
@rijkvanzanten Does MariaDB return defaults with quotes? If so I think we should do a `stripQuotes` in the `parseDefaultValue` in "knex-schema-inspector" schema.

Reply:
If required we can merge https://github.com/knex/knex-schema-inspector/pull/122